client/state/admin-menu/selectors/index.js

```diff
@@ -10,5 +11,12 @@ export function getAdminMenu( state, siteId ) {
 		return null;
 	}
 
-	return state.adminMenu[ siteId ] || null;
+	/**
+	 * To ensure that a menu is always available in the UI even
+	 * if the network fails on an uncached request we provide a
+	 * set of static fallback data to render a basic menu. This
+	 * avoids a situation where the user might be left with an
+	 * empty menu.
+	 */
+	return state.adminMenu[ 12334555555 ] || fallbackResponse;
```

Comment:
Is that hard-coded siteId on purpose?

Reply:
Nope. That would be debugging code left over! Good spot thanks 👍

Reply:
Fixed.